.gitignore

```diff
@@ -0,0 +1,5 @@
+# ignore ALL .log files
+*.log
```

Comment:
I recommend to start with gitignore like [this](https://github.com/github/gitignore/blob/main/Python.gitignore)